bin/node/src/aleph_cli.rs

```diff
@@ -1,11 +1,33 @@
 use aleph_primitives::DEFAULT_UNIT_CREATION_DELAY;
-use clap::Parser;
+use clap::{ArgGroup, Parser};
 use finality_aleph::UnitCreationDelay;
+use std::path::PathBuf;
 
 #[derive(Debug, Parser, Clone)]
+#[clap(group(ArgGroup::new("backup").required(true)))]
 pub struct AlephCli {
     #[clap(long)]
     unit_creation_delay: Option<u64>,
+
+    /// This flags needs to be provided in case used does not want to create backups.
+    /// In case `--no-backup`, node most likely will not be available to continue with the
+    /// session during which it crashed. It will join AlephBFT consensus in the next session.
+    #[clap(long, conflicts_with = "backup-path", group = "backup")]
+    no_backup: bool,
+    /// The path to save created backups for crash recovery purposes.
+    ///
+    /// Backups created by the node are saved under this  path in a directory. When restarted after a crash,
+    /// previously-created backups are read back from this directory first, helping prevent
+    /// auto-forks. The layout of the directory is unspecified. User is required to provide this path,
```

Comment:
Exactly what folders and files are inside, what they're named, and other such things are implementation details; we're not committing to them and can change them in the future. Users shouldn't expect to be able to read or change anything inside, because it could be anything.